controllers/topics/index.js

```diff
@@ -30,6 +30,7 @@ const getSubscribableTopic = require('./getSubscribeableTopic');
 const {getAnonymUser} = require('../../utils/getAnonymUser');
 const UsersFunction = require('../../databases/functions/users');
 const {insertTopics} = require('../../utils/post');
+const {followTopicServiceQueue} = require('../../services/redis');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `followTopicServiceQueue` from the `services/redis` module is correct and doesn't seem to have any issues. However, it's important to ensure that error handling is properly implemented in the `followTopicServiceQueue` function since it's not visible here.
```diff
+ const {followTopicServiceQueue} = require('../../services/redis');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->